### PR TITLE
Document participant pruning restriction per support-7725

### PR DESCRIPTION
**Pull request by oliverse-da**
For technical details, please refer to
https://support.digitalasset.com/s/case/500Pq00000NCXEIIA5/ledger-pruning-unexpected-behavior

### CODE REVIEW

**Review by andreaslochbihler-da — APPROVED** [review]
Thanks @oliverse-da

**Comment by oliverse-da** [comment]
> Comment: Also update other docs versions, if this FAQ entry applies for other versions than 2.10.0.

Thanks Yves. Added.

File: docs/2.10.0/docs/canton/usermanual/pruning.rst [hunk 1]
```diff
@@ -177,6 +177,9 @@ Current Limitations
   it shares a contract. If a participant becomes defunct and stops sending commitments, pruning of the participant will
   not work. Therefore, before you disable a participant, please make sure that is not involved in any contract. This is
   a current limitation.
+- Participants do not allow pruning of state more recent than the most recently observed transaction minus the
+  `ledger-api.max-deduplication-duration` configuration. As a result, an inactive participant may not prune all state
+  older than the configured retention period.
```
Comment (carrielaben-da) [inline docs/2.10.0/docs/canton/usermanual/pruning.rst:182]:
```suggestion
  `ledger-api.max-deduplication-duration` configuration. As a result, an inactive participant may not prune all states
  older than the configured retention period.
```

File: docs/2.7.9/docs/canton/usermanual/pruning.rst [hunk 2]
```diff
@@ -99,6 +99,9 @@ Current Limitations
   it shares a contract. If a participant becomes defunct and stops sending commitments, pruning of the participant will
   not work. Therefore, before you disable a participant, please make sure that is not involved in any contract. This is
   a current limitation.
+- Participants do not allow pruning of state more recent than the most recently observed transaction minus the
+  `ledger-api.max-deduplication-duration` configuration. As a result, an inactive participant may not prune all state
```
Comment (carrielaben-da) [inline docs/2.7.9/docs/canton/usermanual/pruning.rst:103]:
```suggestion
  `ledger-api.max-deduplication-duration` configuration. As a result, an inactive participant may not prune all states
```

File: docs/2.8.10/docs/canton/usermanual/pruning.rst [hunk 3]
```diff
@@ -101,6 +101,9 @@ Current Limitations
   it shares a contract. If a participant becomes defunct and stops sending commitments, pruning of the participant will
   not work. Therefore, before you disable a participant, please make sure that is not involved in any contract. This is
   a current limitation.
+- Participants do not allow pruning of state more recent than the most recently observed transaction minus the
+  `ledger-api.max-deduplication-duration` configuration. As a result, an inactive participant may not prune all state
+  older than the configured retention period.
```
Comment (carrielaben-da) [inline docs/2.8.10/docs/canton/usermanual/pruning.rst:106]:
```suggestion
  `ledger-api.max-deduplication-duration` configuration. As a result, an inactive participant may not prune all states
  older than the configured retention period.
```

File: docs/2.9.4/docs/canton/usermanual/pruning.rst [hunk 4]
```diff
@@ -177,6 +177,9 @@ Current Limitations
   it shares a contract. If a participant becomes defunct and stops sending commitments, pruning of the participant will
   not work. Therefore, before you disable a participant, please make sure that is not involved in any contract. This is
   a current limitation.
+- Participants do not allow pruning of state more recent than the most recently observed transaction minus the
+  `ledger-api.max-deduplication-duration` configuration. As a result, an inactive participant may not prune all state
+  older than the configured retention period.
```
Comment (carrielaben-da) [inline docs/2.9.4/docs/canton/usermanual/pruning.rst:182]:
```suggestion
  `ledger-api.max-deduplication-duration` configuration. As a result, an inactive participant may not prune all states
  older than the configured retention period.
```

**Review by carrielaben-da — APPROVED** [review]
One small grammar fix.

File: docs/2.10.0/docs/canton/usermanual/pruning.rst (hunk 1 above)
Comment (oliverse-da) [reply docs/2.10.0/docs/canton/usermanual/pruning.rst:182]: Thanks Carrie. To use the plural form, I am inclined to change from `state` or `states` to `archived contracts and transactions`. I will make the changes.

**Pull request merged**